jobs/generation/Utilities.groovy

```diff
@@ -1274,6 +1274,74 @@ class Utilities {
     }
 
     /**
+     * Creates the standard job view for a given folder. This does not create the folder.
+     *
+     * @param folderName The folder to create a view for.
+     * @param jobName The name of the job, to use in customizing text. Defaults to folderName if not specified.
+     * @param viewName The name to give the standard view. Defaults to 'Official Builds'.
+     * @param filterRegex The regex that determines what jobs should display in the view.
+     * @return The created view
+     */
+    def static registerFolderView(String folderName, String jobName = null, String viewName = 'Official Builds', String filterRegex = /.*(?<!prtest)$/) {
```

Comment:
On second though, can you change the name of this function to addStandardFolderView?  That's closer to our current naming convention.